ee/src/main/java/org/jboss/as/ee/subsystem/ManagedExecutorServiceAdd.java

```diff
@@ -105,25 +103,26 @@ protected void performRuntime(OperationContext context, ModelNode operation, Mod
         final AbstractManagedExecutorService.RejectPolicy rejectPolicy = AbstractManagedExecutorService.RejectPolicy.valueOf(ManagedExecutorServiceResourceDefinition.REJECT_POLICY_AD.resolveModelAttribute(context, model).asString());
 
         final ManagedExecutorServiceService service = new ManagedExecutorServiceService(name, jndiName, hungTaskThreshold, longRunningTasks, coreThreads, maxThreads, keepAliveTime, keepAliveTimeUnit, threadLifeTime, queueLength, rejectPolicy);
-        final ServiceBuilder<ManagedExecutorServiceAdapter> serviceBuilder = context.getServiceTarget().addService(ConcurrentServiceNames.getManagedExecutorServiceServiceName(name), service);
+        final ServiceBuilder serviceBuilder = context.getCapabilityServiceTarget()
```

Comment:
Unless you want to refactor ManagedExecutorServiceService to support installation as a MSC 1.4.x service, you will need to use:
``` java
ServiceBuilder<ManagedExecutorService> serviceBuilder = context.getCapabilityServiceTarget().addCapability(ManagedExecutorServiceResourceDefinition.CAPABILITY, service);
```

Reply:
done